attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto

```diff
@@ -140,14 +140,14 @@ message Empty {
 
 message AttributeDefinition {
   oneof value {
-    string span_path = 1;
```

Comment:
Don't we need span path?

Reply:
We will eventually, but only once we start using trace client. I'm parking that effort right now to focus on getting the straight projections working at the query layer.